bot/exts/info/pypi.py

```diff
@@ -67,8 +69,14 @@ async def get_package_info(self, ctx: Context, package: str) -> None:
                     log.trace(f"Error when fetching PyPi package: {response.status}.")
 
         if error:
-            await ctx.send(embed=embed, delete_after=INVALID_INPUT_DELETE_DELAY)
-            await ctx.message.delete(delay=INVALID_INPUT_DELETE_DELAY)
+            error_message = await ctx.send(embed=embed)
+            await wait_for_deletion(error_message, (ctx.author.id,), timeout=INVALID_INPUT_DELETE_DELAY)
```

Comment:
I want to know how other feels about this:
```suggestion
            await wait_for_deletion(await ctx.send(embed=embed), (ctx.author.id,), timeout=INVALID_INPUT_DELETE_DELAY)
```
It could degrade readability and make less sense.

Reply:
I considered that but opted not to in the end. If others think it's a good idea though I can commit this.